cirq/sim/simulator.py

```diff
@@ -191,20 +192,21 @@ class SimulatesAmplitudes(metaclass=abc.ABCMeta):
     def compute_amplitudes(
             self,
             program: Union[circuits.Circuit, schedules.Schedule],
-            bitstrings: np.ndarray,
+            bitstrings: Sequence[int],
```

Comment:
This is a silent breaking change. Add code to detect the wrong type being passed in and raise an appropriate exception.

Reply:
I've added the code to `SimulatesIntermediateWaveFunction.compute_amplitudes_sweep`.